### DISCUSSION TITLE
[Suggestion] Some Suggestions For Survival

### DISCUSSION THREAD

**Discussion by xxpowerofsoundxx** [tsunamayo/Starship-EVO#5240] [Suggestions]
I've played a lot of space games. You may have already though of some of these but anyway. 

[Inventory management] This is usually a driver for creativity. Please consider looking at the old voxel game "Starmade" for a good approach. Inventories should be physicalized. Starmade used Storage blocks for this that connected together and dynamically showed how full they were. Cool, useful, and a reason to build larger haulers.

[POI] Hopefully you've thought of this already but planets and space need POIs with loot and enemies. See Empyrion for ideas (Though your game stage and building mechanics already seem 10x better than theirs) The only thing I can recommend taking from them for inspiration are the POIs. The community of builders are already making some amazing assets they've put on the workshop. It would be awesome to use those. 

[Ship Building in Survival] Starmade used actual shipyards that gave you creative mode while in a survival game and you could make a Ghost of a ship that the ship yard could then build as long as it had the access tot he parts. This would be awesome to use here also. Especially with the factory stuff you're putting in. Having it all connect would be amazing.

**Comment by Blodgemeister** [comment]
Definitely some of the better elements from StarMade. I'd wholly support implementing these ideas.
